### DISCUSSION TITLE
How to solve the static resource loading problem of the target server when the target path is dynamic proxy

### DISCUSSION THREAD

**Discussion by ziqingkeshi** [mitre/HTTP-Proxy-Servlet#205] [Q&A] [answered]
Hello,
    I have a requirement that dynamically generated according to a parameter in the request path , Luckily I have done it. But the problem now is when my interception rule :'/proxy/*', target_url :'https://www.example.com/', I can access the homepage normally, but the static resources of the relative path loaded on the homepage ,such as js, css, img are all 404, because these resources requests initiated by them do not carry'/proxy/', so they cannot be intercepted by my server and proxied, what should I do, if you can reply as soon as possible, I will be very grateful!

**Comment by ziqingkeshi** [comment]
@dsmiley

**Comment by dsmiley** [comment, marked as answer]
Assuming I understand your scenario, this is not something the proxy is designed to handle.  Doing this would require rewriting the HTML and maybe other resources coming from the site to insert a leading `/proxy`.  Or add an HTML meta tag that sets the URL base (if I recall; I haven't done front end work in many years).  If you have control over the target site, you could improve it to avoid absolute URLs.  

It's likely you'll have to seek another proxy solution.

**Reply by ziqingkeshi** [reply]
Thank you very much for your reply, I am ready to do some processing on the front end.
